### PR TITLE
Controllers

### PR DESCRIPTION
**Pull request by lisset-orozco**
- [x] Generation of Post´s scaffold and relation of the model User with posts

- [x] Add comments with the gem Annotate

- [x] Modify the view of the home

- [x] Assign the user to the post created

- [x] Create a Post and update the page with Ajax

- [x] Show all posts

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,76 @@
+class PostsController < ApplicationController
+  before_action :set_post, only: [:show, :edit, :update, :destroy]
```
Comment (joalbertg) [inline app/controllers/posts_controller.rb]: use % i or I to array

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+  # GET /posts
+  # GET /posts.json
+  def index
+    @posts = Post.all
+  end
+
+  # GET /posts/1
+  # GET /posts/1.json
+  def show
```
Comment (joalbertg) [inline app/controllers/posts_controller.rb]: in one line

File: app/controllers/posts_controller.rb [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  end
+
+  # GET /posts/new
+  def new
+    @post = Post.new
+  end
+
+  # GET /posts/1/edit
+  def edit
```
Comment (joalbertg) [inline app/controllers/posts_controller.rb]: the same here

File: app/controllers/posts_controller.rb [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  def destroy
+    @post.destroy
+    respond_to do |format|
+      format.html { redirect_to posts_url, notice: 'Post was successfully destroyed.' }
+      format.json { head :no_content }
+    end
+  end
+
+  private
+    # Use callbacks to share common setup or constraints between actions.
```
Comment (joalbertg) [inline app/controllers/posts_controller.rb]: remove indent into lines after the private word

**Review by joalbertg — CHANGES_REQUESTED** [review]

File: app/controllers/posts_controller.rb (hunk 4 above)
Comment (lisset-orozco) [reply app/controllers/posts_controller.rb]: Done!

File: app/controllers/posts_controller.rb (hunk 3 above)
Comment (lisset-orozco) [reply app/controllers/posts_controller.rb]: Done!

File: app/controllers/posts_controller.rb (hunk 2 above)
Comment (lisset-orozco) [reply app/controllers/posts_controller.rb]: Done!

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (lisset-orozco) [reply app/controllers/posts_controller.rb]: Done!

File: app/controllers/posts_controller.rb [hunk 5]
```diff
@@ -1,5 +1,5 @@
 class PostsController < ApplicationController
-  before_action :set_post, only: [:show, :edit, :update, :destroy]
+  before_action :set_post, only: %i[:show :edit :update :destroy]
```
Comment (joalbertg) [inline app/controllers/posts_controller.rb]: this does not work

**Review by joalbertg — CHANGES_REQUESTED** [review]

**Review by joalbertg — APPROVED** [review]

**Pull request merged**